assets/scss/flex.scss

```diff
@@ -212,6 +193,17 @@ Class                   | Variants
 
 }
 
+/*
+Content justification variants are also available (see also: <a class="link" target="_blank" href="https://css-tricks.com/almanac/properties/j/justify-content/">css tricks flexbox demo</a>)
+Using this signature: `.at[Breakpoint]_flex--[justification]`
+
+Class                   | Variants
+----------------------- | -------------------------------
+`.flex--flexEnd`        | justifies content to `flex-end`
```

Comment:
should we be calling these `jusify--[variant]` for clarity? There are a lot of `flex` properties and these only pertain to `justify-content`